plugins/callback/diy.py

```diff
@@ -0,0 +1,1419 @@
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2019, Trevor Highfill <trevor.highfill@outlook.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+  callback: diy
+  callback_type: stdout
+  short_description: Customize the output
```

Comment:
We plan to do a release the coming week, for that we'd need:
```suggestion
  short_description: Customize the output
  version_added: 0.2.0
```